elyra/pipeline/processor_kfp.py

```diff
@@ -287,3 +306,21 @@ def _get_runtime_configuration(self, name):
         except BaseException as err:
             self.log.error('Error retrieving runtime configuration for {}'.format(name), exc_info=True)
             raise RuntimeError('Error retrieving runtime configuration for {}', err) from err
+
+    def _get_user_auth_session_cookie(self, url, username, password):
+        get_response = requests.get(url)
+
+        # auth request to kfp server with istio dex look like '/dex/auth/local?req=REQ_VALUE'
+        if 'auth' in get_response.url:
+            credentials = {'login': username, 'password': password}
+
+            # Authenticate user
+            session = requests.Session()
+            session.post(get_response.url, data=credentials)
+            cookie_auth_key = 'authservice_session'
+            cookie_auth_value = session.cookies.get(cookie_auth_key)
+
+            if cookie_auth_value:
+                return cookie_auth_key + '=' + cookie_auth_value
+
+        return None
```

Comment:
This return (None) statement is not necessary since that's the default behavior in Python.